task_4/app.js

```diff
@@ -0,0 +1,50 @@
+// Create a car object, add a color property to it with the value equals 'black'
+const car = new Object()
+
+// Change the color property of the car object to 'green'
+car.color = 'green'
+
+//Add the power property to the car object, which is a function and displays the engine power to the console
+car.power = function (){ console.log('Horse power: 665')}
```

Comment:
You have added a function that prints a string to the console. Unfortunately, in this form for further development, difficulties may arise, you just need to add a property number, then it can be used more easily.